### PR TITLE
New package: swiften-2.0

### PR DESCRIPTION
**Pull request by jprjr**

### CODE REVIEW

**Comment by jprjr** [comment]
It looks like I still need to figure out how to properly build this

**Comment by jprjr** [comment]
Spruced up the package, this should cross-compile fine now!

File: srcpkgs/swiften/template [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+do_install() {
+	scons \
+	  cc="${CC}" \
+	  cxx="${CXX}" \
+	  linkflags="${LDFLAGS}" \
+	  ccflags="${CXXFLAGS}" \
+	  ${_scons_options} \
+	  SWIFTEN_INSTALLDIR="${DESTDIR}/usr" \
+	  "${DESTDIR}/usr"
+}
```
Comment (xtraeme) [inline srcpkgs/swiften/template]: What's the reason to duplicate what's in `build-style/scons.sh` ?

Comment (jprjr) [inline srcpkgs/swiften/template]: It seems like this package is picky about the names of the variables passed to it - `build-style/scons.sh` uses `CCFLAGS` as the variable name, but it doesn't seem to pick them up unless I use the lowercase `ccflags`. The `build-style/scons.sh` script just passes it in uppercase

**Comment by jprjr** [comment]
Rebased from master to pull in updates to common/shlibs

**Comment by thypon** [comment]
Could you rebase to fix the issues?

**Comment by jprjr** [comment]
Rebased+pushed

**Pull request merged**